connect_ext_ppr/service.py

```diff
@@ -117,6 +118,8 @@ def create_ppr(ppr, context, deployment, db, client, logger):
     new_version = get_ppr_new_version(db, deployment)
     config_kwargs = {}
     status = PPRVersion.STATUS.ready
+    active_configuration = None
+    product_version = None
```

Comment:
[COD-B] Why we have the `product_version` as `None` when user upload a file?, I think the new ppr has to correspond to the product version available at that moment on time no matter where the new file been generated from.

Reply:
As I understand, no.
@d3rky tell us.

Reply:
We cannot guarantee with what version of the product and configuration this PPR was created. That's why it is null. We can assume, that probably it is the latest versions... But who knows? Maybe the version was not released at that moment

Reply:
Sorry was my misunderstanding, then lets proceed!